documents/WCA Bylaws.md

```diff
@@ -20,31 +20,69 @@ The name of this corporation is World Cube Association (“WCA”).
+  1. The Regional Organization has a name, address and contact details;
+  2. The Regional Organization has a website and a logo;
+  3. The Regional Organization provides a current copy of it’s Bylaws to the WCA;
+  4. The Regional Organization is registered as a legal entity in the area in which they operate;
+  5. The Regional Organization provides an description of their organizational structure;
+  6. The Regional Organization provides a clear description of the area they cover along with a report on past activities proving they act as the entity which organises WCA activities within their region.
+This membership is active from the moment they are acknowledged by the Board of Directors as meeting the requirements for this category and continues until this membership is revoked either through resignation or termination. This category of members does not have the right to vote in matters of the WCA and any elections.
+
+### 2.4	Associate Regional Organization Membership
+Associate Regional Organization membership is available for groups who act to organize WCA activities within their specified regions but however, are not currently a registered entity within the region which they operate. Applications for this type of membership are to be made in writing to the Board of Directors provided the entity meets the following requirements:
+  1. The Regional Organization has a name, address and contact details;
+  2. The Regional Organization has a website and a logo;
+  3. The Regional Organization provides a current copy of it’s Bylaws to the WCA;
+  5. The Regional Organization provides an description of their organizational structure;
+  6. The Regional Organization provides a clear description of the area they cover along with a report on past activities proving they act as the entity which organises WCA activities within their region.
+This membership is active from the moment they are acknowledged by the Board of Directors as meeting the requirements for this category and continues until this membership is revoked either through resignation or termination or after two years from the date in which their membership is active. This category of members does not have the right to vote in matters of the WCA and any elections.
+
+### 2.5	Individual Voting Members
+Any individual who is currently a Full Delegate, Committee/Team Leader, Committee/Team Senior Member or a Director is automatically considered an individual voting member of the WCA. This membership is active from the moment an individual is promoted to one of the above positions and continues until this membership is revoked either through resignation or termination (See 2.6 and 2.8). This category of members has the right to place a maximum of one vote in matters of the WCA and any elections.
```

Comment:
According to Motion 2.2020.1, the members with Voting Rights are:
- Director
- Officer
- Committee/Team Leader
- Committee/Team Senior Member
- Senior Delegate
- Regional Delegate
- Full Delegate

So here are missing some of them.

Reply:
Thanks I added officer to the list as I believe that was all that was missing

Reply:
Shouldn't Regional and Senior Delegates also be included? By definition, they are Full Delegates but they are still listed on the mentioned Motion.